fiftyone/core/odm/utils.py

```diff
@@ -657,3 +657,28 @@ class DocumentRegistryError(Exception):
 
 
 _document_registry = DocumentRegistry()
+
+
+def load_dataset(_id=None, name=None):
+    """Loads the dataset from the database by _id or name."""
+    import fiftyone.core.odm as foo
+    import fiftyone.core.dataset as fod
+
+    if name:
+        return fod.Dataset(name, _create=False)
+
+    if not _id:
+        raise ValueError("Must provide either id or name")
+
+    db = foo.get_db_conn()
+    try:
+        uid = ObjectId(_id)
```

Comment:
This has the benefit of allowing the passed `id` to be either a string or `ObjectId`, which is a nice touch 👍